app/assets/templates/layout/navbar_right.html.slim

```diff
@@ -1,12 +1,12 @@
 ul.nav.navbar-nav.navbar-right
   li.my-account
-    a(ng-click="goToMyAccount()") My Account
+    a(ng-click="goToMyAccount()")= I18n.t 'sprangular.my_account'
```

Comment:
Yea, so my idea was to cache templates via ```/assets/templates/en/...```.   I did a quick test running the dummy app in production with assets precompiled, and it appeared to work, but I am distrusting of it.   I wanted to have you look at your caching mechanism to see if you could find a simple solution. As I had a hard time wrapping my head around what was actually going on.